plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -1174,7 +1175,6 @@ public void testEmptySortedByList()
         dropTable(tableName);
     }
 
-
```

Comment:
belongs to ``Support the Iceberg `sorted_by` table property`` commit

(https://github.com/trinodb/trino/pull/15730)